pkg/apis/storage/types.go

```diff
@@ -350,6 +354,16 @@ type CSINodeDriver struct {
 	TopologyKeys []string
 }
 
+// VolumeNodeResources is a set of resource limits for scheduling of volumes.
+type VolumeNodeResources struct {
+	// Maximum number of unique volumes provisioned by the CSI driver that can be used on a node.
+	// A volume that is both attached and mounted on a node is considered to be used once, not twice.
+	// The same rule applies for an unique volume that is shared among multiple pods on the same node.
+	// This field can be nil if there is no limit of volumes supported on a node.
```

Comment:
How about:
If this field is nil, then the supported number of volumes on this node is unbounded

Reply:
Changed